scripts/convert_yaml_to_sdf.py

```diff
@@ -142,25 +182,31 @@ def read_shape_item(shape_item, link_names, color):
     name = unique_name(name, link_names)
     sdf_link_item["name"] = name
 
-    geometry, link_pose, geometry_pose = read_geometry(shape_item)
+    geometry, link_pose, geometry_pose = read_geometry(shape_item, model_name)
 
     # Maybe have a default name for collision and visual instead of link name
     sdf_link_item["collision"] = {"name": name, "geometry": geometry}
```

Comment:
```suggestion
    sdf_link_item["collision"] = {"name": name, "geometry": geometry.copy()}
```
To keep aligned with visual

Reply:
Better aligned, but does use unnecessary extra memory. Not that that really matters for a small dictionary :p

Reply:
True